apps/confidential/confidential_test.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/kylelemons/godebug/pretty"
 )
 
+const localhost = "https://localhost"
```

Comment:
Nit: It is probably irrelevant to this PR, but since you bother refactor all the preexisting `https://localhost` magic strings anyway, do we consider change it to `http://localhost` i.e. without httpS? Most times, localhost test would not have a cert provisioned.